src/components/dashboard/GapChart.vue

```diff
@@ -160,11 +160,11 @@ const spec = computed(() => {
           },
           update: {
             tooltip: {
-              signal: `{
+              signal: `datum.population > 0 ? {
```

Comment:
does this work using the `{signal: '...', test: '...'}` style? that might be more consistent with the rest of the spec if it works

Reply:
Yes! I was able to use the test style!